common/scala/src/main/scala/org/apache/openwhisk/core/entity/WhiskActivation.scala

```diff
@@ -124,6 +124,11 @@ case class WhiskActivation(namespace: EntityPath,
   }
   def withoutLogs = copy(logs = ActivationLogs()).revision[WhiskActivation](rev)
   def withLogs(logs: ActivationLogs) = copy(logs = logs).revision[WhiskActivation](rev)
+
+  def isTimedoutActivation = {
```

Comment:
Can delete curly braces for `isTimedoutActivation` method.

Reply:
We deleted the curly braces for `isTimedoutActivation`, we also found other occurrences in `WhiskActivation` with curly braces (not for us) so we cleaned them too.